client/src/app/services/services-explorer/services-explorer.component.ts

```diff
@@ -20,6 +21,9 @@ export class ServiceExplorerComponent implements OnInit {
 
   isActive(): Boolean {
     return this.user.isBarman() && this.user.barman.active;
+  }
 
+  hasServiceWritePerm(): Boolean {
+    return this.ngxPermissionsService.getPermissions()['service:write'] !== undefined;
```

Comment:
Maybe return directly the boolean: `return !!this.[...];`

Reply:
Yep 👍